framework/integration/test/basic/basic_test.go

```diff
@@ -8,14 +8,19 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/giantswarm/operatorkit/framework/integration/client"
+	"github.com/giantswarm/operatorkit/framework/integration/client/configmap"
 )
 
 // Test_Finalizer_Integration_Basic is a integration test for basic finalizer
 // operations. The test verifies that finalizers are added and removed as
 // expected. It does not cover correct behavior with reconciliation.
+//
+// !!! This test does not work with CRs, the framework is not booted !!!
+//
```

Comment:
Why does this matter here? The test is about the configmap right? Its name can then also be aligned.

Reply:
I mainly did this for myself, so I don't try to run this test with a CR. The test is currently just the configmap though, yes.